pyproject.toml

```diff
@@ -261,6 +261,7 @@ ignore = [
 "tests/**.py" = [  # Some things we want for the moin project are unnecessary in tests.
     "D",  # Ignore docstring rules in tests
     "ANN", # Ignore type annotations in tests
+    "ARG001", # Ignore unused function arguments in tests (typically fixtures)
```

Comment:
if this is only for fixtures I would rather remove it to encourage use of usefixtures https://docs.pytest.org/en/6.2.x/fixture.html#usefixtures